.env.example

```diff
@@ -0,0 +1,11 @@
+DJANGO_SECRET_KEY=mysecretkey
```

Comment:
Create another file
`.env`
in `.env` you store real variables
`.env.example` - empty values and it's public